GoogleForCreators/web-stories-wp
# Replace page layout images with image placeholders

## Summary


Replaces all images in templates with single grid placeholder image. These grids show up in the page layouts pane and when rendering in the editor/final story.

## Relevant Technical Choices


* Switches from `getTemplates` in the `edit-story` api provider to a `getPageLayouts` which does the conversions before loading
* For now, the resource is swapped and centered within the existing placement and dimensions, but the scale of the grid changes depending on the size of the image. This looks not too bad and was what was decided as the v1 functionality of this image replacement.
* Placeholder grid asset is stored in the plugin directory since it is small enough to do so as decided by the team.

## To-do



## User-facing changes



## Testing Instructions


1. Browse page layouts. They should all have grid images replacing normal template images, but should otherwise be identical.
2. Apply a page layout and the image should be the placeholder there as well.

---



Fixes #5833
